src/vision/test/hsv_filter_test.test

```diff
@@ -1,6 +1,13 @@
 <!-- A .test file which launches the node and its corresponding rostest -->
-
 <launch>
-    <node name="hsv_filter_node" pkg="vision" type="hsv_filter_node" />
+    <node name="hsv_filter_node" pkg="vision" type="hsv_filter_node" output="screen">
+            <param name="h_low" value="5" type="int" />
+            <param name="s_low" value="230" type="int" />
+            <param name="v_low" value="230" type="int" />
+            <param name="h_high" value="15" type="int" />
+            <param name="s_high" value="255" type="int" />
+            <param name="v_high" value="255" type="int" />
+        </node>
+        <remap from="chatter" to="hello"/>
```

Comment:
this remap line seems like a copy-paste leftover? I don't think we're using "chatter" or "hello" as topic names in here?